### PR TITLE
More use of standard algorithms

### PR DESCRIPTION
**Pull request by akva2**
Another round of this

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
Other than one transformation that's not strictly equivalent this looks good to me.

File: external/resinsight/ReservoirDataModel/RigWellPath.cpp [hunk 1]
```diff
@@ -115,10 +115,9 @@ const std::vector<double>& RigWellPath::measuredDepths() const
 std::vector<double> RigWellPath::trueVerticalDepths() const
 {
     std::vector<double> tvds;
-    for ( const cvf::Vec3d& point : m_wellPathPoints )
-    {
-        tvds.push_back( std::fabs( point.z() ) );
-    }
+    std::transform(m_wellPathPoints.begin(), m_wellPathPoints.end(),
+                   std::back_inserter(tvds),
+                   [](const auto& point) { return point.z(); });
```
[the remark this answers is not in the document]
Comment (akva2) [reply external/resinsight/ReservoirDataModel/RigWellPath.cpp:120]: thanks. definitely not intended, amended.

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**